src/services/game-logic/pieces/King.ts

```diff
@@ -23,7 +23,11 @@ class King extends Piece {
             row: this.position.row + row,
```

Comment:
do wyrzucenia linia 20 ` const [row, column] = positions;` nie jest to używane nigdzie

Reply:
poprawione